scan/lib/scan/detect_values.rb

```diff
@@ -138,7 +138,7 @@ def self.detect_destination
       elsif Scan.project.tvos?
         Scan.config[:destination] = Scan.devices.map { |d| "platform=tvOS Simulator,id=#{d.udid}" }
       else
-        Scan.config[:destination] = ["platform=OS X"]
+        Scan.config[:destination] = ["platform=macOS"]
```

Comment:
Good point. Can we check the Xcode version there?
I just checked that Xcode 7.3.1 and earlier require "OS X" and Xcode 8 only knows about "macOS".